app/src/main/java/com/sthoray/allright/data/model/shared/Listing.kt

```diff
@@ -0,0 +1,77 @@
+package com.sthoray.allright.data.model.shared
+
+
+import com.google.gson.annotations.SerializedName
+import com.sthoray.allright.data.model.listing.*
+
+/**
+ * Model for listings.
+ *
+ * This model might still be missing a few properties.
+ *
+ * @property id the listing id
+ * @property productName the listing name
+ * @property locationName the listing location
+ * @property startPrice the listing start price
+ * @property buyNowPrice the listing buy now price
+ * @property currentPrice the current auction price
+ * @property shippingType the type of shipping offered
+ * @property shippingOptions a list of [ShippingOption]s
+ * @property mainImage the listing [MainImage]
+ * @property related properties related to that specific listing [Related].
+ * @property storeRelated properties related to the store where the listing came from [StoreRelated].
+ *@property description details of the listing.
+ * @property stats stats about the listing.
+ * @property categories
+ * @property brandTitle for the tile of the brand of the product.
+ * @property outOfStock to check if the product is out of stock.
+ * @property reserveMet to check if the reserve price has met.
+ * @property reservePrice to check if the item has a reserve price.
+ * @property manager
```

Comment:
What is the manager for?

Reply:
"manager": {
    "id": 303,
    "name": "IT-Mate Online",
    "contact_hours": null,
    "primary_phone": null,
    "type": 2,
    "website": "www.it-mate.co.nz",
    "pickups": 0,
    "aggregate_rating": null,
    "award": null,
    "department": null,
    "legal_name": null,